split.js

```diff
@@ -143,6 +156,7 @@ Split = function (ids, options) {
             pair.gutter = gutter;
         }
 
-        el.style.width = 'calc(' + options.widths[i] + '% - ' + options.gutterWidth / 2 + 'px)';
```

Comment:
One thing I see is that the third column of the three-column example slops out of the enclosing <div> even **before** dragging any boundaries. I suspect this is caused by the adjustment to the middle pane's width, to allow for the gutters, is wrong: for the panes on the ends, it is indeed appropriate to adjust by only one-half of gutter width, but for the pane in the middle, the adjustment should be by a **full** gutter width, because it should lose half a gutter to each side.

Without the .toFixed(0), the calculation of pane widths was precise enough that subtracting only 5px from the width of the middle pane, rather than the 10px it should have been, makes the total width -- of all three panes and two gutters -- exceed the width of the containing division. That triggers the slop-over.

Adding the .toFixed(0) makes the default widths of the three panes sum to only 99% rather than 99.9998%. The extra 1% is enough to cover the 5px that should have been subtracted from the middle pane, but were not.